# Spatial Transcriptomics Deconvolution

#### Research Project under Professor Hamim Zafar, Computer Science and Engineering, IIT Kanpur
#### Project Duration: February 2022-Present

## Overview
Benchmarking probabilistic models for spatial mapping of cell types by integrating single cell RNA- Se- quencing and spatial transcriptomic data

## Problem
Spatial Deconvolution can have several highly useful applications in biomedicine. For example, Ccncer is a group of diseases that occurs due to the abnormal and uncontrollable growth of cells and tissue. Several studies have shown that Cancer tissue is highly heterogeneous. This means that cancer tissue has a lot of different types of cells at different locations. The relative locations of the different cell clusters and their function greatly affects the growth of the cancerous tissue. Thus, it would be useful to create a map of the relative locations of the cells and their respective functions. 

## Approach
We will be applying probabalistic models to data obtained by integrating Single Cell RNA sequencing data and Spatial Transcriptomics to identify the relative locations of the cells and their functions, so as to create a cell map. Single Cell RNA sequencing data is used to obtain the knowledge about the function of a cell, but it cannot give information about the location of the cell. On the other hand, Spatial Transcriptomics exclusively captures the location of the cell. Combining these two sets of data, we can create a cell map of the tissue.
Several probabilistic models have been developed for this purpose. For example, “Cell2Location” is one of the most famous algorithms developed for this purpose. Our goal is to compare the performance of several such models and benchmark them. This data will be useful for creating cell maps in the future.

We will also be generating our own datasets to perform benchmarking of the methods. The datasets will be generate using Generative Models. 

